eng/scripts/Language-Settings.ps1

```diff
@@ -147,19 +147,6 @@ function Get-dotnet-AdditionalValidationPackagesFromPackageSet {
     $AllPkgProps
   )
   $additionalValidationPackages = @()
-  $uniqueResultSet = @()
-  function isOther($fileName) {
-    $startsWithPrefixes = @(".config", ".devcontainer", ".github", ".vscode", "common", "doc", "eng", "samples")
-
-    $startsWith = $false
-    foreach ($prefix in $startsWithPrefixes) {
-      if ($fileName.StartsWith($prefix)) {
-        $startsWith = $true
-      }
-    }
-
-    return $startsWith
-  }
 
   # ensure we observe deleted files too
   $targetedFiles = @($diffObj.ChangedFiles + $diffObj.DeletedFiles)
```

Comment:
Is any of this language specific? Is there a any reason it couldn't be in the common function?

Reply:
For `.net` I don't think I have any way to claim that anything custom should handle it. Spending more time in common changes. I'll just delay this PR in favor of getting an eng/common change trhough.

Reply:
Ok all done 😄